QUICKSTART.md

```diff
@@ -1,58 +1,538 @@
+
+For more information, see [Ansible Documentation](http://docs.ansible.com/ansible/intro_installation.html).
+
 
-Run `./tidb-server -h` to see more flag options.
+### 3. Download TiDB-Ansible to the Control Machine
+Download the latest master version of the ZIP package from GitHub [TiDB-Ansible project](https://github.com/pingcap/tidb-ansible) or [click to download]( https://github.com/pingcap/tidb-ansible/archive/master.zip).
+ 
+ 
+### 4. Orchestrate the TiDB cluster
 
-After you started tidb-server, you can use a MySQL client to connect to TiDB.
+The standard Cluster has 6 machines: 
 
+- 2 TiDB instances 
+- 3 PD instances, one of the PD instances is used as the monitor.
+- 3 TiKV instances
+ 
+The cluster topology is as follows:
+
+| Name | Host IP | Services |
+| ---- | ------- | -------- |
+| node1 | 172.16.10.1 | PD1, TiDB1 |
+| node2 | 172.16.10.2 | PD2, TiDB2 |
+| node3 | 172.16.10.3 | PD3, Monitor|
+| node4 | 172.16.10.4 | TiKV1 |
+| node5 | 172.16.10.5 | TiKV2 |
+| node6 | 172.16.10.6 | TiKV3 |
+ 
+Edit the `inventory.ini` file:
+ 
 ```
-mysql -h 127.0.0.1 -P 4000 -u root -D test --prompt="tidb> " 
+[tidb_servers]
+172.16.10.1
+172.16.10.2
+ 
+[pd_servers]
+172.16.10.1
+172.16.10.2
+172.16.10.3
+ 
+[tikv_servers]
+172.16.10.4
+172.16.10.5
+172.16.10.6
+ 
+[monitored_servers:children]
+tidb_servers
+tikv_servers
+pd_servers
+ 
+[monitoring_servers]
+172.16.10.3
+ 
+[grafana_servers]
+172.16.10.3
+
+...
+
 ```
+ 
+### 5. Deploy the TiDB cluster
+
+Use the normal user with the sudo privileges to deploy TiDB:
+	
+5.1 Edit the `inventory.ini` file as follows:
+	
+	 ```
+	  ## Connection
+	  # ssh via root:
+	  # ansible_user = root
+	  # ansible_become = true
+	  # ansible_become_user = tidb
+	  
+	  # ssh via normal user
+	  ansible_user = tidb
+	 ```
+5.2 Connect to the network and download the TiDB, TiKV, and PD binaries:
+		
+		ansible-playbook local_prepare.yml
+	                                           
+5.3 Initialize the system environment of the target machines and adjust the kernel parameters:
+	
+		ansible-playbook bootstrap.yml -k -K
+	 
+**Note:**
+
+- Add the `-k` (lowercase) parameter if password is needed to connect to the managed node. This applies to other playbooks as well.	
+- Add the `-K` (uppercase) parameter if sudo needs password for root privileges.
+	            
+5.4 Deploy the TiDB cluster:
+	 
+	    ansible-playbook deploy.yml -k
+
+### 6. Start the cluster
+	
+Start the TiDB cluster: 
+		
+	ansible-playbook start.yml -k
+
+Use the MySQL client to connect to the TiDB cluster:
+	  	
+	  mysql -u root -h 172.16.10.1 -P 4000
+	  
+**Note:** The TiDB service default port is 4000.
+	  
+
+## Try TiDB
+
+This section describes some basic CRUD operations in TiDB.
+
+### Create, show, and drop a database
+
+You can use the `CREATE DATABASE` statement to create a database. 
+
+The Syntax is as follows:
+
+    CREATE DATABASE db_name [options];
```

Comment:
Try:
```sql
CREATE DATABASE db_name [options];
```
See: https://support.codebasehq.com/articles/tips-tricks/syntax-highlighting-in-markdown